docker/docker-compose.ci.yml

```diff
@@ -21,24 +21,18 @@ services:
       POSTGRES_PASSWORD: '${DB_PASSWORD}'
       POSTGRES_DB: '${DB_NAME}'
   unlock-contract-deployer: # Used for integration tests to deploy the smart contract
-    image: unlock
-    user: node
```

Comment:
this is inside `unlock-core` which is a "parent" image for `unlock-app`. No need to duplicate.